pkgs/games/humblebundle/baba-is-you.nix

```diff
@@ -0,0 +1,27 @@
+{ stdenv, buildGame, fetchHumbleBundle, libGL, makeWrapper }:
+
+buildGame rec {
+  name = "baba-is-you-unstable-${version}";
+  version = "2019-03-31"; # guessed from mar31 in file name.
+
+  # https://dl.humble.com/hempulioy/BIY_linux_mar31.tar.gz
+  src = fetchHumbleBundle {
+    machineName = "babaisyou_odtbg_linux_oR9qb";
+    downloadName = "Download";
+    suffix = "tar.gz";
+    md5 = "61893c3ead0dc1823115ad83dcb9aee0";
+  };
+
+  buildInputs = [ makeWrapper libGL ];
+
+  sandbox.paths.required = [ "$HOME/Baba_Is_You" ];
```

Comment:
Hm, doesn't the game support `XDG_CONFIG_HOME`/`XDG_DATA_HOME`?

Reply:
Could you please add [`libredirect`](https://github.com/NixOS/nixpkgs/tree/8519cd1aa0e0874dedefc1f850cf611f6abc8d57/pkgs/build-support/libredirect) to the wrapper so that the data dir is in `$XDG_DATA_HOME`? The `sandbox.paths.required` directive should already take care of creating the directory, so it should be enough to use something like `NIX_REDIRECTS=$HOME/Baba_Is_You=${XDG_DATA_HOME:-$HOME/.local/share}/Baba_Is_You`.

Reply:
apparently the august update fixed the issue :-)

Reply:
@layus: Ah, didn't see this comment.